app/views/content/life-as-a-teacher/explore-subjects/history/_article.html.erb

```diff
@@ -0,0 +1,73 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach history</h2>
+    <div class="inset">
+      <p>Teaching history, you’ll challenge pupils to scrutinise evidence to build a picture of the past. Guiding them through the often elusive sources left to us, you will help them question the reliability and accuracy of what they read. They’ll build skills in research, handling evidence, and creating arguments.</p>
```

Comment:
Wonder if it should be 'forming arguments', rather than creating? Or forming persuasive arguments?